tests/cmdline/commands/test_export.py

```diff
@@ -213,6 +201,27 @@ def test_migrate_force(self):
                 self.assertTrue(os.path.isfile(filename_output))
                 self.assertEqual(zipfile.ZipFile(filename_output).testzip(), None)
 
+    def test_migrate_in_place(self):
```

Comment:
Please add another test that not specifying an output file and also not the `-i` flag, exist with non-zero

Reply:
Done!